app/Livewire/Boarding/Index.php

```
@@ -320,6 +320,21 @@ public function validateServer()
             return handleError(error: $e, livewire: $this);
         }
 
+        try {
+            // Check prerequisites
+            $prerequisitesInstalled = $this->createdServer->validatePrerequisites();
```

PHPMD (2.15.0)

Avoid excessively long variable names like $prerequisitesInstalled. Keep variable name length under 20. (undefined)

(LongVariable)